### ISSUE TITLE
config extension page doesn't have any extensions

### ISSUE THREAD

**Issue by cdagnino** [ipython-contrib/IPython-notebook-extensions#211]
Hi,

First of all, the config extensions seems like a very neat GUI. I work in different computer and I'm always running into trouble with what extensions they have.

1) I installed the notebook extension repository following [this page](https://github.com/ipython-contrib/IPython-notebook-extensions/wiki/Home_3x), which consisted in running

```python
import IPython.html.nbextensions as nb
ext= 'https://github.com/ipython-contrib/IPython-notebook-extensions/archive/3.x.zip'
nb.install_nbextension(ext)
```
I can load extensions and they work.

2) I followed the instructions in [the config extension page](https://github.com/ipython-contrib/IPython-notebook-extensions/wiki/config-extension). I copied the relevant files and modified the `ipython_notebook_confi.py` file. Running an ipython notebook and going to `/nbextensions/` in the browse leads me to a screen that looks correct: it says "Notebook Extension Configuration" but I don't see any extension. I have them installed, but I suppose the config extension doesn't know where they are.

Maybe I'm missing an obvious step here, thanks!

**Comment by juhasch** [comment]
Not sure what is going on.
You can add a log output to see what is going on in line 37 of `nbextensions.py`. Then you should see a list of all extensions that have been found (actually the .yaml files that describe them).
```Python
        for y in yaml_list:
            self.log.warn(y) # add this line
            stream = open(os.path.join(y[0],y[1]), 'r')
```
If nothing is found here, check if the correct nbextensions directory is used by adding
```Python
 self.log.warn(nbextensions)
```
to line 19 or so.

You will have to restart the notebook server if you change `nbextensions.py`.

**Comment by cdagnino** [comment]
Thanks for the reply.
The first thing didn't show anything. The second one showed this directory

`/Users/username/.ipython/nbextensions`

And that's definitely the problem, because my extensions are not installed there.
I installed the notebook extensions with the code I put above (`nb.install_nbextension(ext)`), but I can't figure out where they got installed! I thought they got dumped inside `.ipython` but that doesn't seem to be the case.

**Comment by JesterEE** [comment]
I fixed this issue on my pull request: cb8a4937b187ba515ce89671ed67517a067d0201

**Comment by juhasch** [comment]
Fixed by #223
